### ISSUE TITLE
Listening ports

### ISSUE THREAD

**Issue by Faeton73** [realies/soulseek-docker#23]
Hi

How do soulseek’s listening ports work in this setting? I have the container running in ubuntu, but when I check for the listening ports via netstat I do not get any results. The only port in use is 6080, which I guess it is just for novnc. Do I need to open any port as I used to do with the Windows client?

Thanks

**Comment by hawk-sor** [comment]
I would also like to know, I am running the docker on Synology's NAS and I cannot check the ports

**Comment by realies** [comment]
Port 6080 should be sufficient.

**Comment by hawk-sor** [comment]
I don't think so, because I still get the message to check if the doors are open when I ask to list the files of some users.  :/

**Comment by jt196** [comment]
@hawk-sor 6080 is the port you're using to access the web interface (as you're probably aware). I believe if you're looking to forward ports on SSQT, check which listening port the system has open (options > login) and forward that on your router. You might need to add the listening port to the docker command too? Think I've had to do that in the past.

**Comment by Enissay** [comment]
> Additional port forwarding might be needed, although this image is tested to work without it. https://www.slsknet.org/news/comment/3874#comment-3874

It indeed seems to be working only with the 6080 port. 
It would be useful thought to give the possibility to set up the app port which seems to be randomized with each run!!
A environment variable would be useful

**Comment by realies** [comment]
> > Additional port forwarding might be needed, although this image is tested to work without it. https://www.slsknet.org/news/comment/3874#comment-3874
> 
> It indeed seems to be working only with the 6080 port. 
> It would be useful thought to give the possibility to set up the app port which seems to be randomized with each run!!
> A environment variable would be useful
> 

App port is not randomized if config is saved. It can't be controlled via an environment variable unless a Soulseek config parser/writer i created.
